stubs/Markdown/markdown/inlinepatterns.pyi

```diff
@@ -53,31 +53,33 @@ class Pattern:
 class InlineProcessor(Pattern):
     safe_mode: bool
     def __init__(self, pattern: str, md: Markdown | None = None) -> None: ...
-    def handleMatch(self, m: re.Match[str], data) -> tuple[Element, int, int] | tuple[None, None, None]: ...  # type: ignore[override]
+    def handleMatch(self, m: re.Match[str], data: str) -> tuple[Element | str | None, int | None, int | None]: ...  # type: ignore[override]
 
 class SimpleTextPattern(Pattern): ...
 class SimpleTextInlineProcessor(InlineProcessor): ...
 class EscapeInlineProcessor(InlineProcessor): ...
 
 class SimpleTagPattern(Pattern):
-    tag: Any
+    tag: str
     def __init__(self, pattern: str, tag: str) -> None: ...
 
 class SimpleTagInlineProcessor(InlineProcessor):
-    tag: Any
+    tag: str
     def __init__(self, pattern: str, tag: str) -> None: ...
 
 class SubstituteTagPattern(SimpleTagPattern): ...
 class SubstituteTagInlineProcessor(SimpleTagInlineProcessor): ...
 
 class BacktickInlineProcessor(InlineProcessor):
-    ESCAPED_BSLASH: Any
+    ESCAPED_BSLASH: str
     tag: str
     def __init__(self, pattern: str) -> None: ...
 
-class DoubleTagPattern(SimpleTagPattern): ...
```

Comment:
This should not have been deleted. It’s [still there](https://github.com/Python-Markdown/markdown/blob/3.5.1/markdown/inlinepatterns.py#L455) upstream.

Reply:
PR welcome!

Reply:
- #11335